### ISSUE TITLE
EA build approach idea

### ISSUE THREAD

**Issue by AdamJGallagher** [wonboyn/traderissues#14]
Would it make sense over time to build every possible indicator into the EA code with exposed parameters whereby the user could switch the indicator on or off in the code and set the parameters for that indicator? I had this idea from an EA called Forex Fury that is kind of like this, with about 25 parameters that the user can set to their preference. It allows for maximum user testing and progressive configuring to get to the point of having optimised settings for different trade strategies - eg momentum, OBOS, swing etc

**Comment by wonboyn** [comment]
I'm not sure. It's certainly achievable. 
I get the idea. My concern is that it makes the EA less intuitive / friendly for the user having a large number of parameters.

**Comment by AdamJGallagher** [comment]
Yes, though the users are just us at this stage. There's probably a sensible balance between 'front of house' and 'back of house' to assist in this testing phase.

**Comment by wonboyn** [comment]
Ok - I'm happy to give it a go. It's simple to expose the parameters & enable toggling on/off. The more complex bit is fitting the on/off into the decision making component. To do that I need instructions for Jeff & yourself on which indicators you want to include & how to use them in open/close decisions
